ocs_ci/ocs/ui/page_objects/alerting.py

```diff
@@ -0,0 +1,247 @@
+import logging
+import re
+import hashlib
+import requests
+from ocs_ci.utility.retry import retry
+from requests import TooManyRedirects, Timeout, HTTPError
+
+from ocs_ci.ocs.ui.page_objects.page_navigator import PageNavigator
+from ocs_ci.ocs.ui.page_objects.searchbar import SearchBar
+
+
+logger = logging.getLogger(__name__)
+
+
+class Runbook:
+    """
+    Runbook object used for checking the runbook content.
+    May be instantiated with the runbook hash value as expected result for the test;
+    If the runbook hash value is not provided, text should be provided and the hash value will be calculated
+    """
+
+    def __init__(self, text=None, runbook_hash=None):
+
+        if not text and not runbook_hash:
```

Comment:
runbook_hash is not mandatory param as per https://github.com/red-hat-storage/ocs-ci/pull/9673/files#diff-1409bcbe58c79d86425176087db7a133b4ffe5fbe2e17134382dec7d6e298b2eR28

Reply:
can't understand the problem here..

Reply:
I gnore. i was confused here